MOBILE_API.xml

```diff
@@ -4685,6 +4846,48 @@
         </param>
     </struct>
 
+    <struct name="SeatStatus" since="7.1">
+      <description>Describes the status of a parameter of seat.</description>
+      <param name="seatLocation" type="SeatLocation" mandatory="true"/>
+      <param name="conditionActive" type="Boolean" mandatory="true"/>
+    </struct>
```

Comment:
```suggestion
    <struct name="SeatStatus" since="7.1">
        <description>Describes the status of a parameter of seat.</description>
        <param name="seatLocation" type="SeatLocation" mandatory="true"/>
        <param name="conditionActive" type="Boolean" mandatory="true"/>
    </struct>
```
Inconsistent spacing in these additions